doc/Getting-started/index.md

```diff
@@ -65,543 +59,261 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-
-The structure of the collected data is flexible and defined by the [log schema***](). Based on the log schema defined for the Kaa application, Kaa generates an object model for the records and the corresponding API calls in the client SDK. Kaa also takes care of data marshalling, managing temporary data storage on the endpoint, and uploading data to the Kaa server.
-
-For programming practice, see [collecting data from endpoints***]().
-
-**Using notifications**
-Kaa uses _notifications_ to distribute structured messages, posted within _notification topics_, from the server to endpoints. A notification structure is defined by a corresponding [notification schema***]().
-
-Endpoint are subscribed to notification topics, which can be either mandatory or optional. Access to notification topics is automatically granted according to the endpoint’s group membership. Notifications can be sent either to every endpoint subscribed to a topic or to an individual endpoint.
-
-Notifications can be assigned expiration timestamps to prevent their delivery after a certain period of time.
-
-For programming practice, see [using notifications***]().
-
-**Distributing operational data**
-Kaa allows you to perform operational data updates, such as configuration data updates, from the Kaa server to endpoints. This feature can be used for centralized configuration management, content distribution, etc. Since Kaa works with structured data and constraint types, it guarantees data integrity.
+# Further reading
 
-The Kaa server monitors the database for changes and distributes updates to endpoints in the incremental form, thus ensuring efficient bandwidth use. The endpoint SDK performs data merging and persistence, as well as notifies the client code about the specific changes made to the data. As a result, the client application knows exactly where in the data structure the changes occurred and can be programmed to react accordingly.
+Use the following guides and references to make the most of Kaa.
 
-Based on the endpoint’s group membership, it is possible to control what data is available to the endpoint. This is achieved by applying group-specific data overrides, which make it possible to adjust the behavior of the client application based on operational conditions or usage patterns, fine-tune the algorithms according to feedback, implement gradual feature roll-out, A/B testing, etc.
-For programming practice, see [distributing data to endpoints***]().
+* **[Key platform features]({{root_url}}Programming-guide/Key-platform-features/)** - 
+Use this reference to learn about features and capabilities of Kaa
+ \( [Endpoint profiling]({{root_url}}Programming-guide/Key-platform-features/Endpoint-profiles/), 
```

Comment:
Endpoint profiles according to https://docs.google.com/spreadsheets/d/1qCYbc4NIHxxuyh1_mMrFUJ1J5yLJLvaxUxA1Na180Ug/edit#gid=640874709